### PR TITLE
Updated Syntax

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
Removed var and replaced with const.
Reference [issue](https://github.com/mdn/content/pull/17820#pullrequestreview-1025798386)
@teoli2003

### CODE REVIEW

File: fetch-request/index.html [hunk 1]
```diff
@@ -19,8 +19,8 @@ <h1>Fetch Request example</h1>
 
   </body>
   <script>
-    var myImage = document.querySelector('img');
-    var myRequest = new Request('flowers.jpg');
+    const myImage = document.querySelector('img');
+    const myRequest = new Request('flowers.jpg');
 
     fetch(myRequest)
     .then(function(response) {
```
Comment (teoli2003) [inline fetch-request/index.html:26]:
```suggestion
    .then( (response) => {
```

File: fetch-request/index.html [hunk 2]
```diff
@@ -30,11 +30,11 @@ <h1>Fetch Request example</h1>
       return response.blob();
     })
     .then(function(myBlob) {
```
Comment (teoli2003) [inline fetch-request/index.html:32]:
```suggestion
    .then((myBlob) => {
```

File: fetch-request/index.html [hunk 3]
```diff
@@ -30,11 +30,11 @@ <h1>Fetch Request example</h1>
...
-      var objectURL = URL.createObjectURL(myBlob);
+      const objectURL = URL.createObjectURL(myBlob);
       myImage.src = objectURL;
     })
     .catch(function(error) {
```
Comment (teoli2003) [inline fetch-request/index.html:36]:
```suggestion
    .catch((error) => {
```

File: fetch-request/index.html [hunk 4]
```diff
@@ -30,11 +30,11 @@ <h1>Fetch Request example</h1>
...
-      var p = document.createElement('p');
+      const p = document.createElement('p');
       p.appendChild(
         document.createTextNode('Error: ' + error.message)
```
Comment (teoli2003) [inline fetch-request/index.html:39]:
```suggestion
        document.createTextNode(`Error: ${error.message}`)
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
As we have to edit both this page and the counterparts on mdn/content each time we want to update this, we should do all the changes in one go.

Here, in addition to `var` to `const`, I have:
- Used arrow functions for anonymous functions
- Used a literal template.

Two more things should be done (but I can't propose change to them in the GitHub UI):

1. Line 10 to 13: the comment for IE9 should be removed.
2. Another literal string should be used at line 28.

**Comment by teoli2003** [comment]
(Note that I don't write access here, so I won't be able to merge this)

**Comment by PrajwalBorkar** [comment]
> As we have to edit both this page and the counterparts on mdn/content each time we want to update this, we should do all the changes in one go.
> 
> Here, in addition to `var` to `const`, I have:
> 
> * Used arrow functions instead of anonymous functions.
> * Used a literal template instead of the concatenation operator.
> 
> Two more things should be done (but I can't propose changes to them in the GitHub UI):
> 
> 1. Line 10 to 13: the comment for IE9 should be removed.
> 2. Another literal string should be used at line 28.

Made the changes 👍

File: fetch-request/index.html [hunk 5]
```diff
@@ -19,24 +14,24 @@ <h1>Fetch Request example</h1>
...
   </body>
   <script>
-    var myImage = document.querySelector('img');
-    var myRequest = new Request('flowers.jpg');
+    const myImage = document.querySelector('img');
+    const myRequest = new Request('flowers.jpg');
 
     fetch(myRequest)
-    .then(function(response) {
+    .then( (response) => {
```
Comment (teoli2003) [inline fetch-request/index.html:21]:
```suggestion
    .then((response) => {
```

**Comment by teoli2003** [comment]
A last typo and this is ready. Pinging @schalkneethling or @rumyra as I don't have the rights to review/merge here.

**Review by schalkneethling — APPROVED** [review]
Thank you, @PrajwalBorkar and welcome to the community 🎉

**Pull request merged**